contract/contracts/Statements.sol

```diff
@@ -0,0 +1,32 @@
+//SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+contract Statements {
+
+    struct Request {
+        uint256 from;
+        uint256 to;
+        bool executed;
+        bytes cid;
```

Comment:
what is it for?

Reply:
cid is IPFS identifier of the file
https://docs.ipfs.io/concepts/content-addressing/ 
so once statements file generated BE would post to IPFS and submit cid on chain for the user to download